apps/builder/app/builder/features/breakpoints/breakpoints-editor.tsx

```diff
@@ -150,15 +150,17 @@ type BreakpointsEditorProps = {
 export const BreakpointsEditor = ({ onDelete }: BreakpointsEditorProps) => {
   const breakpoints = useStore(breakpointsStore);
   const [addedBreakpoints, setAddedBreakpoints] = useState<Breakpoint[]>([]);
-
+  const breakpointsRef = useRef(
+    groupBreakpoints(Array.from(breakpoints.values()))
+  );
```

Comment:
Strange never updating ref. It will always contain `breakpoints` from the first render. Seems like might be intentional, but maybe call it `initialBreakpointsRef` or something?

Reply:
yes, this is how I am preserving the initial sort order so that nothing jumps around